src/functionsShellCommandAction.js

```diff
@@ -13,9 +13,10 @@ var shell = require("./emulator/functionsEmulatorShell");
 
 module.exports = function(options) {
   options.port = parseInt(options.port, 10);
-
   return serveFunctions
-    .start(options, { quiet: true })
+    .start(options, {
```

Comment:
Note: this is the smallest change I could make that makes this correct.  The real issue is that `{ quiet: true }` is not a valid instance of `FunctionsEmulatorArgs` but that gets lost in the JS-->TS change and if I want to fix that I have to modify more files in `serve` that I don't want to touch at the moment.